app/app-message-service.coffee

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+class MessageObject
```

Comment:
I'll take a look on that. I've done it this way because this service is controlling messages on several parts of the same view, that's why I need to create a different object every time a function in the controller need the service.